src/rpc/blockchain.cpp

```diff
@@ -2288,10 +2288,10 @@ UniValue dumptxoutset(const JSONRPCRequest& request)
         }
     }.Check(request);
 
-    fs::path path = fs::absolute(request.params[0].get_str(), GetDataDir());
+    fs::path path = fs::absolute(GetDataDir() / request.params[0].get_str());
```

Comment:
Ah my bad. This is a change in behavior.

Joining "/tmp/" and "/foo" will correctly give:

* `/foo` with the code in current master
* `/foo` with std lib absolute (your version)
* `/tmp//sadf` with the current version

Reply:
Yeah.

Even this https://github.com/bitcoin/bitcoin/pull/19466/files#diff-e802a36c28b0140bab62cb5366199656R97 breaks multiwallet test.

So the PR without those changes is essentially empty and it looks to me I can close it.

Reply:
> So the PR without those changes is essentially empty and it looks to me I can close it.

Jup, was my fault for suggesting to split them up